.github/workflows/sync_wiki.yml

```diff
@@ -0,0 +1,75 @@
+name: Synchronize wiki pages
+
+################################################################################
+#
+#   🚨  🚨  🚨
+#
+#   If you edit this file, be sure to make the same edit in all of the other
+#   repos whose wikis should participate in synchronization. The GitHub App we
+#   use to synchronize the repos themselves does not have permission to sync
+#   workflow files, so this file (and others in .github/workflows) must be
+#   manually copied.
+#
+################################################################################
+
+on:
+  gollum:
```

Comment:
I didn't know about gollum until I googled it five seconds ago, so possibly a link here? https://github.com/gollum/gollum